Makefile

```diff
@@ -6,7 +6,8 @@ SHELL := bash
 MAKEFLAGS += --warn-undefined-variables
 MAKEFLAGS += --no-builtin-rules
 MAKEFLAGS += --no-print-directory
-BIN=.tmp/bin
```

Comment:
https://go.dev/doc/go1.19#os-exec-path - We have to change the PATH to allow execution in go 1.19+.